source/guide_prosody.rst

```diff
@@ -0,0 +1,393 @@
+ certificates = "/home/isabell/etc/certificates/"
+ https_certificate = "/home/isabell/etc/certificates/upload.isabell.uber.space.crt"
+ http_ports = { }
+ https_ports = { FILEUPLOAD-PORT }
+
+ ----------- Virtual hosts -----------
+ VirtualHost "isabell.uber.space" 
+ Component "conference.isabell.uber.space" "muc"
+   modules_enabled = { "muc_mam", "vcard_muc" }
+ Component "upload.isabell.uber.space" "http_upload" 
+   http_upload_file_size_limit = 10485760
+   http_upload_expire_after = 2419200
+   http_upload_path = "/home/isabell/prosody/http_upload/"
+
+.. warning:: Replace the placeholders ``C2S-PORT``, ``S2S-PORT`` and ``FILEUPLAD-PORT`` with the above obtained ports, adapt the domain-names, sql settings (inclusive username and password) and paths! Don't delete, obmit or change the ordering of the entries, otherwise some default ports could be spammed. Also don't active modules which including module ``http`` without changing ``http_ports`` and ``https_ports`` . Last but not least be warned that spamming the default ports which could already be in use can lead to fork-spam issues! So be careful and watch your configuration twice and look into the prodoy logs afterwards to verify whats going on after starting prosody! 
+
+Setup deamon
+============
+
+Place the file ``prosody.ini`` in ``~/etc/services.d/`` and adapt it accordingly:
+
+.. code-block:: ini
+
+ [program:prosody]
+ command=/bin/bash -c "source %(ENV_HOME)s/.bash_profile && prosody"
+ autostart=yes
+ autorestart=yes
+ startretries=1
+ stopasgroup=true
+ killasgroup=true
+ stopsignal=INT
+
+.. include:: includes/supervisord.rst
+
+Finishing installation
+======================
+
+Create your first user:
+
+.. code-block:: console
+ :emphasize-lines: 2,3 
+
+ [isabell@stardust ~]$ prosodyctl adduser name@isabell.uber.space
+ Enter new password:
+ Retype new password:
+ [isabell@stardust ~]$
+
+Updates
+=======
+
+.. note:: Check the `update feed`_ regularly to stay informed about the newest version.
+
+For updates simply repeat the steps described in the Installation_ part:
+
+::
+
+ [isabell@stardust ~]$ cd ~/prosody
+ [isabell@stardust ~]$ wget https://prosody.im/downloads/source/prosody-0.XX.X.tar.gz
+ [...]
+ [isabell@stardust ~]$ tar -xzf prosody-0.XX.X.tar.gz
+ [isabell@stardust ~]$ cd ~/prosody/prosody-0.XX.X
+ [isabell@stardust ~]$ ./configure --ostype=linux --prefix=$HOME --with-lua-include=/usr/include
+ Lua version detected: 5.1
+ Lua interpreter found: /usr/bin/lua...
+ Checking Lua includes... lua.h found in /usr/include/lua.h
+ Checking if Lua header version matches that of the interpreter... yes
+ Writing configuration...
+
+ Installation prefix: /home/isabell
+ Prosody configuration directory: /home/isabell/etc/prosody
+ Using Lua from: /usr
+
+ Done. You can now run 'make' to build.
+
+ [isabell@stardust ~]$ make
+ [...] 
+ [isabell@stardust ~]$ make install
+ [...] 
+ [isabell@stardust ~]$ supervisorctl restart prosody
+ [isabell@stardust ~]$ 
```

Comment:
Let's make this section Copy&Paste-friendly (in combination with the other suggestion).
```suggestion
 [isabell@stardust ~]$ VERSION=X.XX.X
 [isabell@stardust ~]$ cd ~/prosody
 [isabell@stardust prosody]$ wget https://prosody.im/downloads/source/prosody-$VERSION.tar.gz
 [...]
 [isabell@stardust prosody]$ tar -xzf prosody-$VERSION.tar.gz
 [isabell@stardust prosody]$ cd ~/prosody/prosody-$VERSION
 [isabell@stardust prosody-X.XX.X]$ ./configure --ostype=linux --prefix=$HOME --with-lua-include=/usr/include
 Lua version detected: 5.1
 Lua interpreter found: /usr/bin/lua...
 Checking Lua includes... lua.h found in /usr/include/lua.h
 Checking if Lua header version matches that of the interpreter... yes
 Writing configuration...

 Installation prefix: /home/isabell
 Prosody configuration directory: /home/isabell/etc/prosody
 Using Lua from: /usr

 Done. You can now run 'make' to build.

 [isabell@stardust prosody-X.XX.X]$ make
 [...] 
 [isabell@stardust prosody-X.XX.X]$ make install
 [...] 
 [isabell@stardust prosody-X.XX.X]$ supervisorctl restart prosody
 [isabell@stardust prosody-X.XX.X]$ 
```